libvast_test/src/table_slices.cpp

```diff
@@ -324,8 +319,9 @@ void table_slices::test_message_serialization() {
   auto sink = make_sink();
   CHECK_EQUAL(sink(slice1), caf::none);
   MESSAGE("load content for the second slice from the buffer");
-  auto source = make_source();
-  CHECK_EQUAL(source(slice2), caf::none);
+  // FIXME: legacy_deserializer does not handle caf::message
+  // CHECK_EQUAL(vast::detail::legacy_deserialize(buf, slice2), true);
+  return; // FIXME: Not necessary
```

Comment:
Just remove this test; it's covered by other tests anyways and not very meaningful.

Reply:
Resolved in c0311cfd71c165a71bc0e8c4f941fa938572e14d